src/Alert.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Store from './store'
+import { connect } from 'react-redux'
+import AlertActions from './actions/alerts'
+
+class Alert extends React.Component {
+  componentWillReceiveProps(nextProps) {
+    if(nextProps.alert !== this.props.alert) setTimeout(() => Store.dispatch(AlertActions.reset()), 5000)
```

Comment:
I found this non-trivial. Should we add a comment explaining what this line does?